packages/perps-exes/assets/config-price.toml

```diff
@@ -2089,3 +2089,27 @@ inverted = false
 [networks.rujira-testnet.ATOM_USDC.feeds-usd.constant]
 price = "1"
 inverted = false
+
+[[networks.rujira-testnet.ATOM_USD.feeds]]
```

Comment:
Let's stick with ATOM_USDC instead of ATOM_USD. ATOM_USD would use ATOM as the collateral asset, ATOM_USDC uses USDC, which matches our intended mainnet deploy.

Reply:
Then you're trying to add the wrong market. Look at help output, it's defaulting to that market.